.dockerignore

```diff
@@ -0,0 +1 @@
+Gemfile.lock
```

Comment:
Shouldn't we base on the lock file?

Reply:
`Gemfile.lock` is not in the repo and it should not be, as for gems, you don't want to require specific versions of dependencies (or so I was told https://stackoverflow.com/questions/4151495/should-gemfile-lock-be-included-in-gitignore). I added it to `.dockerignore` to make sure the locally generated file does not get into the Docker build.

Happy to discuss this.

Reply:
OK this kept bugging me, so I read a bit deeper and it seems reasonable to check in `Gemfile.lock` file after all, even though this is a gem.

See this comment https://stackoverflow.com/questions/4151495/should-gemfile-lock-be-included-in-gitignore#comment75432623_4151540 and this answer https://stackoverflow.com/a/38964304/4603021 on the same question. I tend to agree with these.

If that's OK, I'll create a separate PR to add the `Gemfile.lock` to the repo and remove it from `.dockerignore`.

Reply:
Thats fine 👍